src/main/java/net/swedz/extended_industrialization/item/RobotAutoFeederItem.java

```diff
@@ -65,7 +67,7 @@ public long getEnergyMaxOutput(ItemStack stack)
 	
 	public boolean hasEnergy(ItemStack stack)
 	{
-		return this.getStoredEnergy(stack) > 0;
+		return this.getStoredEnergy(stack) >= EAT_ENERGY_COST;
```

Comment:
I would prefer that this change is done in a separate PR so it is clearer in the generated change log that multiple changes were made